client/package.json

```diff
@@ -34,7 +34,7 @@
     "contributes": {
         "configuration": {
             "type": "object",
-            "title": "Example configuration",
+            "title": "Yseop Markup Language Configuration",
```

Comment:
Use the same capitalization rules as other configs. → “configuration” (OK for Yseop Markup Language because YML, though)

Reply:
Done.